### ISSUE TITLE
Accessibility: use of colour in cards

### ISSUE THREAD

**Issue by gaktive** [rancher/dashboard#4380]
When looking at accessibility per rancher/dashboard#4324, we have a problem with colour usage is on “cards” with the top line colour coded. This page does not provide additional information to explain the difference in card colours (the line on top that can be blue, yellow or red), only after opening the dropdown  

Below, you will find a screenshot for quick reference: 
 
![colour1](https://user-images.githubusercontent.com/11466788/137369162-973b951c-e71a-406d-b897-ef592d87b3df.png)

Even if the dropdown was always open, the criteria would still fail since users that depend on SR would not be able to read the references. 

![colour2](https://user-images.githubusercontent.com/11466788/137369183-dd54b130-be01-4d71-a227-faeef1cc80c1.png)

Suggested solution: 
- Add an `aria-label` with the description associated description to each of the cards. For example, a yellow card should have the `aria-label=”Partner providers/services/cluster”`. We recommend that there is more context than just the words “rancher”, “partners”, or “RKE2” so the user understands exactly to what they refer. This lines up with an overall suggestion to rely on an aria-label in each `<div>` to describe the category type for assistive technologies.

**Comment by lvuch** [comment]
Related:
https://xd.adobe.com/view/66c03ec6-d686-44f5-a6b6-ed33e6975f90-3393/
https://github.com/rancher/dashboard/issues/5007

**Comment by gaktive** [comment]
Can't do this until #5007 is done

**Comment by kwwii** [comment]
This is part of the accessibility improvement list and is ready for implementation (add aria-labels as appropriate throughout the ui)

**Comment by kwwii** [comment]
This require a new card design which takes into account ALL possible information which should be included in cards.

Off the top of my head, this includes: Icon, Name, Version, Description, OS, Beta-status, and Type/Source

**Comment by kwwii** [comment]
This is solved in the redesign done by Edén.

**Comment by nwmac** [comment]
Closing this - this will be covered by the new card design for 2.12
